### ISSUE TITLE
Automatiser la création des CSV

### ISSUE THREAD

**Issue by AntoineAugusti** [etalab/inventaire-codes-sources-organismes-publics#4]
Voir tous les deux comment automatiser ceci, sûrement avec CircleCI.

J'ai expliqué le principe récemment dans un article de blog : https://blog.antoine-augusti.fr/2018/12/collecter-et-constituer-automatiquement-et-gratuitement-un-jeu-de-donnees-open-data/

**Comment by bzg** [comment]
Oui - et voir si ça vaut la peine de le faire ou si nous basculons tout de suite sur la construction (automatisée) d'une base de données avec une API fournissant du `.json`.

**Comment by AntoineAugusti** [comment]
Petit PoC d'API sans hébergement et avec déploiement automatique toutes les 3h : https://github.com/AntoineAugusti/api-codes-sources-fr

**Comment by AntoineAugusti** [comment]
Et un répertoire qui automatise la récupération des infos (2 fois / heure) en CSV et JSON des répertoires et organisations : https://github.com/AntoineAugusti/data-codes-sources-fr. Il y a un fichier `all.{csv,json}` pour chaque.

L'API citée avant se sert du coup de ces données.

Je pense qu'à partir de là on verra ce qu'on fait après une discussion de vive voix @bzg.

**Comment by bzg** [comment]
Oui. Je suis en congé jusqu'au 7 janvier, c'est une super base pour discuter. Merci !
